cmd/podman-mac-helper/uninstall.go

```diff
@@ -54,5 +54,23 @@ func uninstall(cmd *cobra.Command, args []string) error {
 	if err := os.RemoveAll(helperPath); err != nil {
 		return fmt.Errorf("could not remove helper binary path: %s", helperPath)
 	}
+
+	// Get the file information of dockerSock
+	fi, err := os.Lstat(dockerSock)
+	if err != nil {
+		// Return an error if unable to get the file information
+		return fmt.Errorf("could not stat dockerSock: %v", err)
+	}
+	// Check if dockerSock is a symbolic link
+	if fi.Mode()&os.ModeSymlink == 0 {
+		return fmt.Errorf("dockerSock is not a symbolic link")
+	}
+	// Attempt to remove dockerSock
+	if err := os.Remove(dockerSock); err != nil {
+		if !os.IsNotExist(err) {
```

Comment:
use `!errors.Is(err, fs.ErrNotExist)` for checking please

Reply:
fixed